services/validation-service.js

```diff
@@ -408,7 +408,7 @@ class ValidationService {
         const segments = ual.split(':');
         const argsString = segments.length === 3 ? segments[2] : `${segments[2]}:${segments[3]}`;
         const args = argsString.split('/');
-        if (!(args?.length === 3)) throw Error('Invalid UAL.');
+        if (!(args?.length === 3 || args?.length === 4)) throw Error('Invalid UAL.');
```

Comment:
Is this because of the state in UAL? 
If so, I was also thinking about this but isn't the state delimited by another `:` and **not** `/` ? @Mihajlo-Pavlovic

Reply:
It's new ual with did:dkg:blockchain/contract/collectionId/assetId

Reply:
oh ok I did not know about this; so its  did:dkg:blockchain/contract/collectionId/assetId:state ? (optional state, other is required?)

Reply:
No assetId is also optional